### ISSUE TITLE
Feature Suggestion: Ability to Change "Today I am grateful for" 

### ISSUE THREAD

**Issue by deepydarkness** [alisonthemonster/GratitudeJournal#173]
Been loving this app and using it since last November. Thanks for your work and all that you do!

One feature I'd like to see is the option to change the "I am grateful for" header that displays at the top of every entry. My entries usually start with "I am grateful I got to...", so if that was already the header it would really help me type down my thoughts more quickly. I think a lot of people can benefit from this feature since different people have different writing styles. Some even use this app for purposes other than gratitude journaling.

If this is difficult to implement (idk cuz I don't code), it's also fine if you could offer "pre-written" headers that we can choose from, instead of the option for us to type down our own customized headers.

Here's the header I'm talking about:
![ASD](https://user-images.githubusercontent.com/72245447/108181446-e47b5400-7142-11eb-866d-0faae144a509.jpg)

**Comment by alisonthemonster** [comment]
Hi there! Thanks for the feature request! Just want to clarify a few things:
- You want to be able to change the text, "I am grateful for" to another custom sentence, would this apply to every entry or be customizable per entry?
- Right now the title "I am grateful for" is not included into your entry itself, is that something you want to happen? You want the title phrase to automatically add itself to your entry?

**Comment by deepydarkness** [comment]
>     * You want to be able to change the text, "I am grateful for" to another custom sentence, would this apply to every entry or be customizable per entry?

I was only thinking of applying it to every entry, but per-entry customization could also be a useful feature to some people.

>     * Right now the title "I am grateful for" is not included into your entry itself, is that something you want to happen? You want the title phrase to automatically add itself to your entry?

Oh yes this is something I'd want happen! As well as having the option to change both title phrase and auto-entry phrase from "I am grateful for" into "I'm grateful I got to". Would really speed up my writing.

**Comment by peakvalleytech** [comment]
Hi, i'd just like to chime in. I was thinking about this issue and know a great implementation. 
You could include a 'default' prompt somewhere like the settings. And allow the prompt to be editable for each entry. 
That would elegantly handle both features.

**Comment by alisonthemonster** [comment]
@peakvalleytech that sounds great!

**Comment by Parasasaniasandip** [comment]
Yes, This is great feature so user can use same app for different purposes.

For example to control anger.

So in settings user may pre-defined few (like 5) header.

When user click for entry then first default header should come. (I.e. Today, I am gratitude for) on click header itself the header content shoud rotate from settings pre-defined header.
 
--------------------

The plan is analysis my iratation like..

Today, I am irritating for (Header)

(Content, I may write)
What makes me irritating?
How i responded?
What is the parmanent solution?

-------------------
Even, We can make it batter in settings you can provide each header wise different colours and allow same day multiple entry
So user can write same day one entry for gratitude and one for anger management.

I think i wrote a lot feature, if you can m grateful if not even existing app is awasome and i not find any similar in app Store. 

For long time i maintained two poket diary now one is fit in my mobile.

Thank you so much developer for such a great app without ad and simplicity.

Best wishes !

**Comment by rakeshprasanna** [comment]
I want to work on this

**Comment by rishujam** [comment]
Hey, can I work on this?
